### PR TITLE
Introduce object equality

### PR DESCRIPTION
**Pull request by fkoehler**
Addresses #2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 target/
 project/project/
 project/target/
+.idea
```
Comment (kolotaev) [inline .gitignore:4]: Actually, this is considered a bad practice. Such things as IDE's, dev-env, OS, etc. specific files should be placed into `.gitignore_global`. The motivation behind it - it easy becomes a nightmare to support all the files for all the possible IDEs etc.
Library gitignore file should contain files related only to the library itself and language/platform it's written in.

**Review by kolotaev — APPROVED** [review]

**Comment by kolotaev** [comment]
@fkoehler, thanks for the contribution!

File: .gitignore (hunk 1 above)
Comment (fkoehler) [reply .gitignore:4]: I see. I am happy if your remove it.

**Comment by fkoehler** [comment]
Can we have a release for this change?

**Comment by kolotaev** [comment]
@fkoehler, sure! I was going to do that, but first I wanted to take a deeper look at the SecureRandom issue we discussed so that include both fixes into the next release, but was a little bit busy.
I can release 1.0.1 with only object equality fix included and later on consider SecureRandom for the next one. Does it fit you?

**Comment by fkoehler** [comment]
That would be perfectly fine for us. Thanks.

Am 20.02.2018 9:10 nachm. schrieb "Egor Kolotaev" <notifications@github.com

**Comment by kolotaev** [comment]
@fkoehler, the new version 1.0.1 is now released. The artifact is on JCenter. As always, use
```
resolvers += Resolver.jcenterRepo
libraryDependencies += "com.github.kolotaev" %% "ride" % "1.0.1"
```

**Comment by fkoehler** [comment]
Awesome, thanks a lot

Am 20.02.2018 9:34 nachm. schrieb "Egor Kolotaev" <notifications@github.com

**Pull request merged**